bitcoin-magazine/bitcoin-explained/bitcoin-core-24.0.md

```diff
@@ -0,0 +1,1150 @@
+So there's already some nodes doing it.
+Now there's more, presumably.
+
+## Merchants
+
+Aaron van Wirdum: 00:21:06
+
+So does this mean that some merchants are going to be unhappy?
+
+
+Sjors Provoost: 00:21:12
+
+Well, they might be for that reason.
+
+
+Aaron van Wirdum: 00:21:13
+
+There are still some merchants that rely on zero-conf.
+
+Sjors Provoost: 00:21:15
+
+Yeah, there are definitely merchants that do that.
+But the flip side is that there are also merchants that use Lightning and Lightning is generally bothered a lot by this, by the existing RBF rules because they make it much more complicated to deal with penalty transactions.
+So they may get, in the long run, they may get a better lightning experience out of this.
+
+Aaron van Wirdum: 00:21:36
+
+I mean, my personal opinion is that this should just be the general rule.
+Transactions should always be broadcast and forwarded, especially if they have a higher fee.
+
+Sjors Provoost: 00:21:48
+
+I mean, maybe, we've already had this discussion during that episode, I guess, but I would also say that if you just turn on the RBF flag by default, it shouldn't bother you either.
+But I think the bigger problem is that this RBF flag has very specific rules.
+It's not just that you set the flag, you also have to do a few other things and those other things cause complications.
+So that could be a reason to say, you know what, let's forget about this opt-in RBF completely and relay everything that is reasonable.
+But there are trade-offs.
+One is just bandwidth because if my node relays everything that you sent to it, I could send you one transaction and then increase it by one satoshi per byte or increase it by.
+Don't even increase the fee, but just change the destinations a bit, and I could send you millions and millions of variations of the same transaction, basically wasting everybody's bandwidth.
+So it's not entirely without trade-offs.
+
+## Descriptor Wallet Migration
+
+
+Aaron van Wirdum: 00:22:39
+
+All right, let's move on to the third point.
+So we mentioned the change on the peer-to-peer network, the block header thing, and we just mentioned RBF.
+And then the third point is this related to descriptor wallets and migration to the descriptor wallets?
+
+Sjors Provoost: 00:22:52
+
+Yeah, I'm not sure how much we covered the descriptor wallets so far but the gist of it is that the Bitcoin Core wallet is quite old.
+
+It used to be just a bag of keys and then given a private key, the wallet would have to pay attention to certain scripts.
+I think in one of the first episodes we explained what scripts are.
+
+But the simplest scripts is just that anybody with the public key can spend this coin.
+The second simplest script is that anybody with the hash of the public key, like, you know, can spend the coin or anybody with the, you know, the public key for which the hash is blah, blah, blah.
```

Comment:
Here ",like, you know," is an example of a filler word/phrase - eliminating it helps with readability
```suggestion
The second simplest script is that anybody with the hash of the public key, can spend the coin or anybody with the, the public key for which the hash is blah, blah, blah.
```